src/ws/cockpitwebservice.c

```diff
@@ -1410,6 +1415,33 @@ cockpit_web_service_get_creds (CockpitWebService *self)
   return self->creds;
 }
 
+/**
+ * cockpit_web_service_get_int:
+ * @self: the service
+ *
+ * Returns: The logging id of this service.
```

Comment:
s/logging/login/, or perhaps better "XDG session ID"?

Reply:
I changed it to "The id of this service, for logging."